notifier/senders/mail/mail.go

```diff
@@ -30,18 +31,24 @@ type Sender struct {
 	dateTimeFormat string
 }
 
-// Init read yaml config
-func (sender *Sender) Init(senderSettings map[string]string, logger moira.Logger, location *time.Location, dateTimeFormat string) error {
+// NewSender creates Sender instance.
```

Comment:
New<Something> seems th be the conventional way to name constructor in golang, thus these commants are redundant

Reply:
Lets wait for answer from Xenia, maybe we will remove it